packages/atomic/src/components/result-template-components/atomic-result-text/atomic-result-text.tsx

```diff
@@ -29,6 +29,11 @@ export class AtomicResultText {
    */
   @Prop() public shouldHighlight = true;
 
+  /**
+   * The non-localized text to display if the field has no value.
```

Comment:
I think `locale key` is clearer compared to `non-localized text`. If we provide a default, we can mention this too.

```
The locale key for the text to display when the configured field has no value.

@default "noTitle"
```